service/access.go

```diff
@@ -347,6 +371,13 @@ func (a *accessService) updateAccessToken(ctx context.Context, domain, role, pro
 	return at.(*AccessTokenResponse), err
 }
 
+func accessCacheMemoryUsage(acd *accessCacheData) int64 {
+	structSize := int64(unsafe.Sizeof(acd.token) + unsafe.Sizeof(acd.domain) + unsafe.Sizeof(acd.role) + unsafe.Sizeof(acd.proxyForPrincipal) + unsafe.Sizeof(acd.expiresIn) + unsafe.Sizeof(acd.expiry) + unsafe.Sizeof(acd.scope))
```

Comment:
can simplify like this, example: https://go.dev/play/p/dhxoE5zsbye
```suggestion
	structSize := int(unsafe.Sizeof(*accessCacheData))
```

Reply:
@WindzCUHK fixed in https://github.com/AthenZ/athenz-client-sidecar/pull/18/commits/f7ee09262991a3e783a5d792984b29cec285ccc0